entrypoint.sh

```diff
@@ -18,7 +18,11 @@ if ! grep ${SSH_USER} /etc/passwd ; then
 
 	# Copy the authorized_keys
 	touch /home/${SSH_USER}/.ssh/authorized_keys
-	echo "$SSH_KEY" >> /home/${SSH_USER}/.ssh/authorized_keys
+	IFS=',' read -a keys <<< "${SSH_KEY}"
+    	for key in "${keys[@]}"
```

Comment:
@JesusZapata Every time we start the ssh container the keys will be added again (restart the server, updating containers) so will end with a bunch of duplicated keys, don't we?

Why not delete the file if exists or ```>``` with the first key? ( I think it is easier to delete it)